example/javalib/testing/5-test-stealing/foo/test/src/foo/RandomTestsA.java

```diff
@@ -0,0 +1,20 @@
+package foo;
+
+import org.junit.Test;
+
+public class RandomTestsA extends RandomTestsUtils {
+  @Test
+  public void test1() throws Exception {
+    testGreeting("Storm", 38);
+  }
+
+  @Test
+  public void test2() throws Exception {
+    testGreeting("Bella", 25);
+  }
+
+  @Test
+  public void test3() throws Exception {
+    testGreeting("Cameron", 32);
+  }
```

Comment:
Do we really need more than one test per file? If not we could consolidate each file to a single test case to greatly reduce the verbosity of these test examples